### PR TITLE
Replaced bleach with strip_tags.

### PR DESCRIPTION
**Pull request by aspidites**
Tests take forever to run as-is, do we really want to generate 500+ contact records just to test this, or is it fine as is?

### CODE REVIEW

**Comment by tdphillips** [comment]
What is the likelihood that a contact record contains something that resembles but is not an HTML tag? The docs for strip_tags claim it just looks for something enclosed in angle brackets.

**Comment by aspidites** [comment]
I assume rare and that such a use case is something that would be turned up during the week of qc.

**Comment by tdphillips** [comment]
Good enough for me.

:+1:, anyone else?

File: mypartners/views.py [hunk 1]
```diff
@@ -1157,7 +1152,7 @@ def prm_export(request):
                 ', '.join([val.name for val in v.all() if val]) for v in values
             ]
             # Remove the HTML and reformat.
-            values = [bleach.clean(v, [], strip=True) for v in values]
+            values = [strip_tags(v) for v in values]
```
Comment (JLMcLaugh) [inline mypartners/views.py]: Is this marked safe on the front end? strip_tags isn't secure, but if we're escaping, then it should be fine.

**Pull request merged**